src/components/Location.react.js

```diff
@@ -106,7 +116,7 @@ export default class Location extends Component {
         this.updateLocation(this.props);
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) {
+    componentWillReceiveProps(nextProps) {
```

Comment:
We want to keep the `UNSAFE_` prefix - unless you can figure out how to use a different (safe) method.

Reply:
Hmm, that's strange, I don't even remember removing that. Might be a side-effect of formatting? 

In any case, re-added in c0e72b7. Thanks for pointing it out!

Reply:
I'd guess a mis-merge... or perhaps you had the file open already when you pulled the latest `dev`. Anyway it's fixed now, thanks!